src/code/z_vr_box.c

```diff
@@ -1,186 +1,239 @@
+#include "z64skybox.h"
 #include "global.h"
 #include "buffers.h"
 
-u32 D_801C5E30[] = { 0, 0x2000, 0x4000, 0x6000, 0x8000, 0xC000 };
+typedef struct SkyboxFaceParams {
+    /* 0x00 */ s32 xStart;
+    /* 0x04 */ s32 yStart;
+    /* 0x08 */ s32 zStart;
+    /* 0x0C */ s32 outerIncrVal;
+    /* 0x10 */ s32 innerIncrVal;
+} SkyboxFaceParams; // size = 0x14
 
-u16 D_801C5E48[] = {
+// Converts texture coordinate values to s10.5 fixed point
+#define TC(x) ((s16)((x)*32))
```

Comment:
Still would like to use this macro in a lot more places, basically anywhere that uses vtx manually, but then that can be addressed in a later PR when I eventually introduce vtx.h/z64vtx.h with those vtx macros that were discussed during the file-select PR. So can leave this for now